pkgs/development/python-modules/matplotlib/default.nix

```diff
@@ -15,32 +16,35 @@ assert enableTk -> (tcl != null)
                 && (tkinter != null)
                 && (libX11 != null)
                 ;
+assert enableQt -> pyqt4 != null;
 
 buildPythonPackage rec {
   name = "matplotlib-${version}";
-  version = "1.5.3";
+  version = "2.0.0";
 
   src = fetchurl {
     url = "mirror://pypi/m/matplotlib/${name}.tar.gz";
-    sha256 = "1g7bhr6v3wdxyx29rfxgf57l9w19s79cdlpyi0h4y0c5ywwxr9d0";
+    sha256 = "04zqymd5dw6lxvfbxf1sycdnibjk5qky5rfsn6wb46lwha2hkkrn";
   };
 
   NIX_CFLAGS_COMPILE = stdenv.lib.optionalString stdenv.isDarwin "-I${libcxx}/include/c++/v1";
 
   XDG_RUNTIME_DIR = "/tmp";
 
-  buildInputs = [ python which sphinx stdenv ]
+  uildInputs = [ python which sphinx stdenv ]
```

Comment:
I’ve fixed and re-pushed it.

I am (re) running `nox-review` to be sure but that might take some time before I have rebuilt everything (I’ve have had to `nix-collect-garbage` since I did my previous testing)